.ci/generate_bear_requirements.py

```diff
@@ -189,7 +198,10 @@ def deep_diff(target, src):
 
 
 def sort_requirements(req_dict):
-    for key in ['pip_requirements', 'npm_requirements', 'gem_requirements']:
+    for key in ['pip_requirements',
```

Comment:
store this list in a global variable structured ready for the next entry to be added as a single line modification.